snapcraft/internal/meta/_manifest.py

```diff
@@ -18,16 +18,20 @@
 import os
 import json
 from collections import OrderedDict
-from typing import Any, Dict  # noqa: F401
+from typing import Any, Dict, TYPE_CHECKING
 
 import snapcraft
 from snapcraft.internal import errors, os_release, steps
 from snapcraft.internal.states import GlobalState, get_state
 
+if TYPE_CHECKING:
+    from snapcraft.project import Project
 
-def annotate_snapcraft(data, parts_dir: str, global_state_path: str):
+
+def annotate_snapcraft(project: "Project", data: Dict[str, Any]) -> Dict[str, Any]:
     manifest = OrderedDict()  # type: Dict[str, Any]
     manifest["snapcraft-version"] = snapcraft._get_version()
+    manifest["snapcraft-started-at"] = project._get_start_time().isoformat() + "Z"
```

Comment:
I don't like the way Python deals with times without timezones, but it's not something we can change right now.